### ISSUE TITLE
Pagination is rendered incorrectry

### ISSUE THREAD

**Issue by vanyakosmos** [geex-arts/django-jet#435]
There are 26 pages of events, and pagination looks like this:

<img width="194" alt="Screenshot 2020-02-19 at 13 21 45" src="https://user-images.githubusercontent.com/16745500/74829998-dd94e800-531a-11ea-847e-10c2b916cbd8.png">

Django version:
```
Django==2.2.10
django-jet==1.0.8
```

With django 2.1 everything is fine

**Comment by akamoroz** [comment]
Hello @vanyakosmos ,
If it this issue is still actual for you please check the small patch:
```
\diff --git a/jet/static/jet/js/src/layout-updaters/paginator.js b/jet/static/jet/js/src/layout-updaters/paginator.js
index 02e6dcb..fb309b7 100644
--- a/jet/static/jet/js/src/layout-updaters/paginator.js
+++ b/jet/static/jet/js/src/layout-updaters/paginator.js
@@ -16,7 +16,7 @@ PaginatorUpdater.prototype = {
             if (($node.prev().prop('tagName') == 'A' || $node.prev().prop('tagName') == 'SPAN')
                 && ($node.next().prop('tagName') == 'A' || $node.next().prop('tagName') == 'SPAN')) {
 
-                if ($.trim($node.text()) == '...') {
+                if ($.trim($node.text()) == 'â€¦') {
                     $node.wrap($('<span>').addClass('disabled'));
                 } else if ($.trim($node.text()) == '') {
                     $node.remove();
--
```

**Comment by vanyakosmos** [comment]
@akamoroz Thanks. Changing that line to `if (['...', '…'].includes($.trim($node.text()))) {` solved the problem. But I will keep the issue open until it fixed in jet (if ever)

**Comment by Alessyo** [comment]
I am unable to fix the error with the line of code above. Do you need to do anything else?

**Comment by vanyakosmos** [comment]
yeah, you will also need to rebuild js bundle after making any changes in js files. Run `gulp scripts`

**Comment by petersenpeter** [comment]
Does anybody have a proper solution to this issue?

**Comment by simwoo7** [comment]
I solved this issue with changing admin_list.py.

`…  -> ...`

@register.simple_tag
def paginator_number(cl, i):
    """
    Generate an individual page index link in a paginated list.
    """
    if i == DOT:
        return **'...' <-- change this**
    elif i == cl.page_num:
        return format_html('<span class="this-page">{}</span> ', i + 1)
    else:
        return format_html(
            '<a href="{}"{}>{}</a> ',
            cl.get_query_string({PAGE_VAR: i}),
            mark_safe(' class="end"' if i == cl.paginator.num_pages - 1 else ''),
            i + 1,
        )
